src/web/shared/Styles.ts

```diff
@@ -600,10 +600,17 @@ export function getRallyStyle(rallyFeatureCount: number) {
  */
 export function divePacketIconStyle(feature: Feature, animatedColor?: string) {
     // Depth text
-    let text = feature.get('depthAchieved') ? feature.get('depthAchieved').toFixed(1) : null
-    if (text != null) {
-        text = text + 'm'
-    } else {
+    const taskPacket = feature.get('taskPacket') as TaskPacket
+    const dive = taskPacket.dive as DivePacket
+    if (dive == null) {
+        return null
+    }
+
+    let text
+    if (dive.bottom_dive == true && dive.depth_achieved != null) {
```

Comment:
We can be more explicit here (=== true) and (!== undefined). Or we can shorten it to if (dive.bottom_dive && dive.depth_achieved).

Reply:
The shorter one will return false if depth_achieved is 0, though.  Going with the first option here.